docs/source/AdministratorGuide/InstallingDIRACService/index.rst

```diff
@@ -226,169 +229,169 @@ required DIRAC functionality. The SystemAdministrator interface can be used late
-        Systems += RequestManagement
-        Systems += ResourceStatus
-        Systems += StorageManagement
-        Systems += Transformation
-        Systems += WorkloadManagement
-        #
-        # List of DataBases to be installed (what's here is a list for a basic installation)
-        Databases = InstalledComponentsDB
-        Databases += ResourceStatusDB
-        #
-        #  The following options define components to be installed
-        #
-        #  Name of the installation host (default: the current host )
-        #  Used to build the URLs the services will publish
-        #  For a test installation you can use 127.0.0.1
-        # Host = dirac.cern.ch
-        #  List of Services to be installed (what's here is a list for a basic installation)
-        Services  = Configuration/Server
-        Services += Framework/ComponentMonitoring
-        Services += Framework/SystemAdministrator
-        Services += ResourceStatus/ResourceStatus
-        #  Flag determining whether the Web Portal will be installed
-        WebPortal = yes
-        WebApp = yes
-        #
-        #  The following options defined the MySQL DB connectivity
-        #
-        # The following option define if you want or not install the mysql that comes with DIRAC on the machine
-        # InstallMySQL = True
-        Database
-        {
-          #  User name used to connect the DB server
-          User = Dirac # default value
-          #  Password for database user acess. Must be set for SystemAdministrator Service to work
-          Password = XXXX
-          #  Password for root DB user. Must be set for SystemAdministrator Service to work
-          RootPwd = YYYY
-          #  location of DB server. Must be set for SystemAdministrator Service to work
-          Host = localhost # default, otherwise a FQDN
-          Port = 3306 # default, otherwise the port
-          #  There are 2 flags for small and large installations Set either of them to True/yes when appropriated
-          # MySQLSmallMem:        Configure a MySQL with small memory requirements for testing purposes
-          #                       innodb_buffer_pool_size=200MB
-          # MySQLLargeMem:        Configure a MySQL with high memory requirements for production purposes
-          #                       innodb_buffer_pool_size=10000MB
-        }
+        #  User name used to connect the DB server
+        User = Dirac # default value
+        #  Password for database user acess. Must be set for SystemAdministrator Service to work
+        Password = XXXX
+        #  Password for root DB user. Must be set for SystemAdministrator Service to work
+        RootPwd = YYYY
+        #  location of DB server. Must be set for SystemAdministrator Service to work
+        Host = localhost # default, otherwise a FQDN
+        Port = 3306 # default, otherwise the port
+        #  There are 2 flags for small and large installations Set either of them to True/yes when appropriated
```

Comment:
Deviating from the scope of this PR, a comment here on MySQL conf. In v7r0 we want to use DIRACOS by default, and DIRAC does not contain MySQL. Should we remove this part related to MySQL installation and configuration?